### ISSUE TITLE
I.L. FE: Incorrect accordion header levels on "Your income limits" page

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#14593]
## Description
On the last page of the Income Limits app, where the user sees all the accordions with various income level options, the accordion header levels are incorrect for the page hierarchy. They are currently H2s but they should be H3s.

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/171af0d6-61e5-4675-907d-d10b80122d8f)

### User story

**AS A** Veteran using assistive technology
**I WANT** the heading hierarchy of the page to be properly defined
**SO THAT** I can easily understand the page structure and navigate directly to the section relevant to me.

## Acceptance criteria

- [ ] Each income level accordion has a heading level of H3
- [ ] Accessibility lead review

**Comment by wesrowe** [comment]
@randimays, here's a small a11y ticket for I.L. 2-pointer?

**Comment by randimays** [comment]
@wesrowe I thought the smallest dev story (requiring a PR) would be 3 points? Either way, it's pretty small. 2 or 3 works.

**Comment by randimays** [comment]
I have a PR open for this: https://github.com/department-of-veterans-affairs/vets-website/pull/25110 but the review instance isn't working yet. I'll have to take another look at this on Wednesday. If we need to get this across the line before I get back, please feel free to take action on it.

**Comment by laflannery** [comment]
The review instance is working now however I can't get through the app in order to get to the last page - the zip code screen isn't allowing me to get past, it's saying that valid zip codes are all invalid:

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/f1d168ac-cf5a-44a4-a8b2-d59fdb3d0f65)

This is also happening on the PR that @chriskim2311 had me review for the auto-scrolling

Tagging @wesrowe because I wasn't sure if this was something he should/could take a look at as well?

**Comment by laflannery** [comment]
Chris explained to me that this was never going to work in the review instance. I looked at Randi's screenshot instead and this is approved from an accessibility standpoint

**Comment by randimays** [comment]
Verified this is working as expected in Staging. Closing.

<img width="917" alt="Screenshot 2023-08-09 at 4 36 17 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/c6810e9f-dde1-4461-8126-8f8960097e61">
